src/redux/categories/Cat.js

```diff
@@ -0,0 +1,12 @@
+import Header from '../../components/Header';
+
+function Cat() {
+  return (
+    <div className="Cat">
+      <Header />
+      <button type="button">Check status</button>
+    </div>
+  );
+}
+
+export default Cat;
```

Comment:
Similary, please remove the `Cat.js` component in the Redux directory. It unnecessary to have it in the Redux file structure. Kindly move it to the `components` directory.

- Redux should only contain reducers for a particular component, but not the component itself.

Reply:
Implemented 👍